tests/mail/tests.py

```diff
@@ -2145,8 +2367,16 @@ def setUpClass(cls):
             port=port,
         )
         cls._settings_override = override_settings(
-            EMAIL_HOST=cls.smtp_controller.hostname,
-            EMAIL_PORT=cls.smtp_controller.port,
+            EMAIL_PROVIDERS={
+                "default": {
+                    "BACKEND": "django.core.mail.backends.smtp.EmailBackend",
+                    "OPTIONS": dict(
+                        settings.EMAIL_PROVIDERS["default"]["OPTIONS"],
+                        host=cls.smtp_controller.hostname,
+                        port=cls.smtp_controller.port,
+                    ),
+                },
+            }
```

Comment:
Copying in the default OPTIONS should be unnecessary here. (If it *is* somehow necessary, that invalidates these tests, because users won't be copying in the default OPTIONS in their own settings.)

```suggestion
            EMAIL_PROVIDERS={
                "default": {
                    "BACKEND": "django.core.mail.backends.smtp.EmailBackend",
                    "OPTIONS": dict(
                        host=cls.smtp_controller.hostname,
                        port=cls.smtp_controller.port,
                    ),
                },
            }
```